### PR TITLE
Admin Installation Steps

### PR DESCRIPTION
**Pull request by justinegeffen**
Added installation steps for admins.

@itao: I wasn't able to test the cloud activation steps; if you could validate that would be awesome.

### CODE REVIEW

File: docs/administrator-guide/installation-and-activation.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+3. Select **Install**.
+4. Next, select **Configure**.
+5. Select **true** to enable the plugin.
+6. Select **Save**.
+
+When you open the Main Menu, **Playbooks & Incidents** is available as a menu item.
+
+## Activating Mattermost Incident Management for Cloud Deployments
+
+Mattermost Incident Management is included in the Mattermost Cloud workspace but needs to be activated prior to use.
```
Comment (itao) [inline docs/administrator-guide/installation-and-activation.md:16]: Every Cloud account comes with this feature enabled by default.

**Comment by itao** [comment]
@justinegeffen Cloud accounts come with it installed and enabled by default so no action is required by the user.

File: docs/administrator-guide/installation-and-activation.md (hunk 1 above)
Comment (justinegeffen) [inline docs/administrator-guide/installation-and-activation.md:16]:
```suggestion
Mattermost Incident Management is included in the Mattermost Cloud workspace and is enabled by default.
```

**Review by itao — APPROVED** [review]

**Comment by justinegeffen** [comment]
@cpoile, if you could give this an approving review that'd be great. :)

**Review by cpoile — APPROVED** [review]

**Pull request merged**